packages/ui/src/containers/SocialLinkAccount/index.tsx

```diff
@@ -16,12 +19,48 @@ type Props = {
   relatedUser: SocialRelatedUserInfo;
 };
 
+const getCreateAccountContent = (
+  signUpMethods: string[]
+): { desc: TFuncKey; buttonText: TFuncKey } => {
+  if (
+    signUpMethods.includes(SignInIdentifier.Email) &&
+    signUpMethods.includes(SignInIdentifier.Phone)
+  ) {
+    return {
+      desc: 'description.social_link_phone_or_email',
+      buttonText: 'action.link_another_email_or_phone',
```

Comment:
it'll be better to keep the same order of email and phone here

Reply:
updated